pageserver/src/tenant/timeline.rs

```diff
@@ -4316,7 +4308,7 @@ impl Timeline {
             if l.get_lsn_range().end > horizon_cutoff {
                 debug!(
                     "keeping {} because it's newer than horizon_cutoff {}",
-                    l.filename().file_name(),
+                    l.filename(),
```

Comment:
```suggestion
                    l,
```

And this would work right? Or embedding it into the message as `{l}` which I cannot suggest.

I think this would be the better way of letting everyone know "aah you can just `{layer}`", instead of reaching for any longer paths.

Reply:
The Display impl of Layer is not the same as calling `.filename()`, at least not in all cases. It *is* the same as calling `.short_id` though, at least in the places that this PR is migrating. It might cause changes in the logs.

Reply:
Hmmm actually wait, you are right, I remember `short_id` calls `filename` internally. I can migrate that too i guess.

Reply:
I think it would be worthwhile for the "teaching others" perspective, and hopefully getting rid of multiple ways to format a layer.

Reply:
Oki, this seems to be all I could find as well.